### ISSUE TITLE
CBus "lights" detected but turning them on/off does nothing

### ISSUE THREAD

**Issue by rollsch** [micolous/cbus#45]
![image](https://github.com/micolous/cbus/assets/1109554/b13e4dac-0555-4cfd-86a0-7a97ae95154c)

Everything detects as the above. I also get the binary sensor equivalent as well. However nothing actually turns on or off.

Any assistance? I managed to get a log stream somehow (cannot find it again and found this

`2023-12-27 WARNING (MainThread) [homeassistant.components.mqtt.mixins] MQTT device name is equal to entity name in your config {'name': 'C-Bus Light 254', 'unique_id': 'cbus_light_254', 'command_topic': 'homeassistant/light/cbus_254/set', 'state_topic': 'homeassistant/light/cbus_254/state', 'schema': 'json', 'brightness': True, 'device': {'identifiers': ['cbus_light_254'], 'connections': [['cbus_group_address', '254']], 'sw_version': 'cmqttd https://github.com/micolous/cbus', 'name': 'C-Bus Light 254', 'manufacturer': 'Clipsal', 'model': 'C-Bus Lighting Application', 'via_device': 'cmqttd'}, 'brightness_scale': 255, 'flash_time_long': 10, 'payload_not_available': 'offline', 'payload_available': 'online', 'encoding': 'utf-8', 'enabled_by_default': True, 'qos': 0, 'color_temp': False, 'xy': False, 'retain': False, 'optimistic': False, 'hs': False, 'rgb': False, 'effect': False, 'availability_mode': 'latest', 'white_scale': 255, 'color_mode': False, 'flash_time_short': 2}, this is not expected. Please correct your configuration. The entity name will be set to `null``

**Comment by Nik19792000** [comment]
I don't think this log is the culprit. I'd suggest to install MQTT Explorer and check what's happening when you turn on lights via cbus switch VS turning them on via home assistant.

**Comment by rollsch** [comment]
Great suggestions, I was looking for a sniffer of some sort. I'll try that and report back. Thank you,.

**Comment by rollsch** [comment]
Ok I install MQTT explorer and I see nothing when I switch the lights via the CBUS panel switches. I tried sending commands to various groups etc, but nothing happens.

Any ideas how I can log the CBUS switches and see what they are sending, then implement them manually in MQTT

**Comment by rollsch** [comment]
Interestingly in C-Bus Toolkit it shows everything as "Off" regardless of whether its off or not, however the CBUS system is working perfectly.

Any ideas what is going on here? We have soft start on most of the lights, but at least half are just relay on off.

Problem may be that the network says "closed" and it thinks it needs serial, however Ive managed to connect to everything via ethernet.

**Comment by rollsch** [comment]
![WhatsApp Image 2023-12-31 at 16 34 19_cd0539e2](https://github.com/micolous/cbus/assets/1109554/48c611f7-127f-4ada-99a1-af5318855fef)
![WhatsApp Image 2023-12-31 at 16 34 19_e85e523c](https://github.com/micolous/cbus/assets/1109554/f94ce4c6-0c2f-48ba-8b68-38695e86a2d5)
![WhatsApp Image 2023-12-31 at 16 34 20_2ace84d9](https://github.com/micolous/cbus/assets/1109554/230d2a69-ab9e-4f99-a666-00af67477e79)
![WhatsApp Image 2023-12-31 at 16 34 20_a5bf3071](https://github.com/micolous/cbus/assets/1109554/9bf64965-786f-4699-8369-350942710f69)
![WhatsApp Image 2023-12-31 at 16 34 21_4a79ad2b](https://github.com/micolous/cbus/assets/1109554/55103c63-e14b-4b26-8ff6-0d62a1d331d7)

This is the controller we have, maybe it doesn't actually support what I need ?

It looks like the project is just the default project.

NMAP shows the following ports open on the IP 

Nmap scan report for 192-168-68-110.tpgi.com.au (192.168.68.110)
Host is up (0.00027s latency).
Not shown: 997 closed tcp ports (reset)
PORT     STATE SERVICE          VERSION
135/tcp  open  msrpc?
139/tcp  open  netbios-ssn      Microsoft Windows netbios-ssn
8081/tcp open  blackice-icecap?

**Comment by rollsch** [comment]
Looks like you can't actually program the touch screen units via ethernet despite having an ethernet port. You have to use another module that does USB -> ethernet etc.

**Comment by Nik19792000** [comment]
At least I haven't seen this kind of unit and therefore don't know what it exposes to the network.
The library is meant for serial interface (PCI) or network interface (CNI). According to documentation, the latter would listen on port 10001.
Without the right connectivity, libcbus won't be able to translate between MQTT and CBUS.

**Comment by rollsch** [comment]
Thanks for your help. Where would be the best place to ask/research what kind of adapter/translater I need to convert the touchscreen interface to PCI/CNI ?
